packages/plugin-commands-outdated/src/outdated.ts

```diff
@@ -173,6 +181,7 @@ export async function handler (
       manifest: await readProjectManifestOnly(opts.dir, opts),
     },
   ]
+  checkPkgInDependencies(packages, args, opts.dir)
```

Comment:
This won't work when the args are patterns. This check should be done in `@pnpm/outdated`

Reply:
https://github.com/pnpm/pnpm/blob/1a1a8bef7a59b084d86052c629a781200f368370/packages/outdated/src/outdatedDepsOfProjects.ts#L25-L30

in `@pnpm/outdated` it always ends up checking each package's dependencies and I can't group all dependencies together